brainscore_language/benchmarks/syntaxgym/__init__.py

```diff
@@ -62,10 +62,23 @@ def __init__(self, suite_ref_list):
             SyntaxGymSingleTSE(suite_ref) for suite_ref in suite_ref_list]
 
     def __call__(self, candidate: ArtificialSubject) -> Score:
-        final_score = Score()
-        final_score.values = np.mean([sub_benchmark(candidate) for sub_benchmark in self.sub_benchmarks])
+        sub_scores = []
+        for sub_benchmark in self.sub_benchmarks:
+            sub_score = sub_benchmark(candidate)
+            sub_score = sub_score.expand_dims('sub_benchmark')
+            sub_score['sub_benchmark'] = [sub_benchmark.suite.meta["name"]]
+            sub_scores.append(sub_score)
+
+        # DEV
+        from pprint import pprint
+        pprint(sub_scores)
+
+        sub_scores = Score.merge(*sub_scores)
```

Comment:
`AttributeError: 'numpy.ndarray' object has no attribute 'rename'`

This doesn't work at the moment and I don't understand why. Can someone with `xarray` experience help out here?
All the `sub_scores` are `DataArray` values.

There's a corresponding test `brainscore_language/benchmarks/syntaxgym/test.py:test_score_aggregate` you can use to test this yourself. Here's the current test output:

```
=========================================================================================== FAILURES ===========================================================================================
_____________________________________________________________________________________ test_score_aggregate _____________________________________________________________________________________
                                                                                                                                                                                                
distilgpt2 = <brainscore_language.model_helpers.huggingface.HuggingfaceSubject object at 0x7ff0b0cec0d0>                                                                                        
                                                                                                                                                                                                
    def test_score_aggregate(distilgpt2):                                                                                                                                                       
        suites = ["cleft", "cleft_modifier"]                                                                                                                                                    
        benchmark = SyntaxGymTSE(suites)                                                                                                                                                        
                                                                                                                                                                                                
>       score = benchmark(distilgpt2)                                                                                                                                                           
                                                                                                                                                                                                
brainscore_language/benchmarks/syntaxgym/test.py:1155:                                                                                                                                          
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
brainscore_language/benchmarks/syntaxgym/__init__.py:77: in __call__                                                                                                                            
    sub_scores = Score.merge(*sub_scores)
../../miniconda3/envs/brainscore/lib/python3.10/site-packages/brainscore_core/metrics/__init__.py:97: in merge
    raise e
../../miniconda3/envs/brainscore/lib/python3.10/site-packages/brainscore_core/metrics/__init__.py:90: in merge
    attr_values = Score.merge(*attr_values, ignore_exceptions=True)
../../miniconda3/envs/brainscore/lib/python3.10/site-packages/brainscore_core/metrics/__init__.py:85: in merge
    result = merge_data_arrays(scores)
../../miniconda3/envs/brainscore/lib/python3.10/site-packages/brainio/assemblies.py:322: in merge_data_arrays
    merged = xr.merge((data_array.rename('z') for data_array in data_arrays))['z'].rename(None)
../../miniconda3/envs/brainscore/lib/python3.10/site-packages/xarray/core/merge.py:895: in merge
    for obj in objects:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

.0 = <tuple_iterator object at 0x7ff0aef8c7c0>

>   merged = xr.merge((data_array.rename('z') for data_array in data_arrays))['z'].rename(None)
E   AttributeError: 'numpy.ndarray' object has no attribute 'rename'

../../miniconda3/envs/brainscore/lib/python3.10/site-packages/brainio/assemblies.py:322: AttributeError
------------------------------------------------------------------------------------- Captured stdout call -------------------------------------------------------------------------------------
[<xarray.Score (sub_benchmark: 1)>
array([0.25])
Coordinates:
  * sub_benchmark  (sub_benchmark) <U5 'cleft'
Attributes:
    raw:      [False False  True  True False False False False False False Fa...,
 <xarray.Score (sub_benchmark: 1)>
array([0.35])
Coordinates:
  * sub_benchmark  (sub_benchmark) <U14 'cleft_modifier'
Attributes:
    raw:      [False  True False  True False  True False False False  True  T...]
```

Reply:
just pushed a fix, see if that works. You will need to also update `core` for it to work.